src/services/refactor.service.js

```diff
@@ -0,0 +1,9 @@
+export const refactorPhoneNumber = (phone = '') => {
+  if (typeof phone !== 'string') return '';
```

Comment:
Please, make this `if` multiline because in current formatting it is not so readable

Reply:
fixed